src/auth/auth-api-request.ts

```diff
@@ -320,9 +319,9 @@ export class FirebaseAuthRequestHandler {
    * Looks up a user by uid.
    *
    * @param {string} uid The uid of the user to lookup.
-   * @return {Promise<Object>} A promise that resolves with the user information.
+   * @return {Promise<object>} A promise that resolves with the user information.
```

Comment:
Does the linter complain about javadoc types too?

Reply:
Not really, but I thought it makes sense to keep the jsdoc types in sync with the implementation. WDYT?

Reply:
The downside is that it will not align with our externs format. Could this be an issue?

Reply:
I think that's fine since the externs are curated by hand anyway. Going forward we can adhere to the distinction:
* Only Typescript in source (everything on Github)
* Only JS in API docs (externs)